Cannot create releases

### Confirmation

- [x] I have checked for similar issues.

### Describe the bug

The "Release" step of the "Release" pipeline is broken with this error.

```shell
   ⨯ release failed after 3m1s               
    error=
    │ build failed: exit status 1: # github.com/shirou/gopsutil/disk
Error:     │ ../../../go/pkg/mod/github.com/shirou/gopsutil@v3.21.4-0.20210419000835-c7a38de76ee5+incompatible/disk/disk_openbsd.go:59:53: cannot use stat.F_mntfromname[:] (value of type []byte) as []int8 value in argument to common.IntToString
Error:     │ ../../../go/pkg/mod/github.com/shirou/gopsutil@v3.21.4-0.20210419000835-c7a38de76ee5+incompatible/disk/disk_openbsd.go:60:51: cannot use stat.F_mntonname[:] (value of type []byte) as []int8 value in argument to common.IntToString
Error:     │ ../../../go/pkg/mod/github.com/shirou/gopsutil@v3.21.4-0.20210419000835-c7a38de76ee5+incompatible/disk/disk_openbsd.go:61:52: cannot use stat.F_fstypename[:] (value of type []byte) as []int8 value in argument to common.IntToString
Error:     │ ../../../go/pkg/mod/github.com/shirou/gopsutil@v3.21.4-0.20210419000835-c7a38de76ee5+incompatible/disk/disk_openbsd.go:149:45: cannot use stat.F_fstypename[:] (value of type []byte) as []int8 value in argument to common.IntToString
    target=openbsd_amd64_v1
Error: The process '/opt/hostedtoolcache/goreleaser-action/2.9.0/x64/goreleaser' failed with exit code 1
```

### Expected behaviour

The release pipeline should pass.

### Environment

Version: <!-- Hemi Network v1.2.3 -->
Operating System: <!-- e.g. Windows 11, macOS 14.2.1, Ubuntu 24.04 -->
Architecture: <!-- amd64, arm64 -->


### Additional Information

Link to the failure: https://github.com/hemilabs/heminetwork/actions/runs/15187757809/job/42712641900#step:10:116

I can reproduce with:

```shell
GOOS=openbsd GOARCH=amd64 go build ./service/tbc/
# github.com/shirou/gopsutil/disk
../../../go/pkg/mod/github.com/shirou/gopsutil@v3.21.4-0.20210419000835-c7a38de76ee5+incompatible/disk/disk_openbsd.go:59:53: cannot use stat.F_mntfromname[:] (value of type []byte) as []int8 value in argument to common.IntToString
../../../go/pkg/mod/github.com/shirou/gopsutil@v3.21.4-0.20210419000835-c7a38de76ee5+incompatible/disk/disk_openbsd.go:60:51: cannot use stat.F_mntonname[:] (value of type []byte) as []int8 value in argument to common.IntToString
../../../go/pkg/mod/github.com/shirou/gopsutil@v3.21.4-0.20210419000835-c7a38de76ee5+incompatible/disk/disk_openbsd.go:61:52: cannot use stat.F_fstypename[:] (value of type []byte) as []int8 value in argument to common.IntToString
../../../go/pkg/mod/github.com/shirou/gopsutil@v3.21.4-0.20210419000835-c7a38de76ee5+incompatible/disk/disk_openbsd.go:149:45: cannot use stat.F_fstypename[:] (value of type []byte) as []int8 value in argument to common.IntToString
```

--- a/database/tbcd/level/upgrade.go
+++ b/database/tbcd/level/upgrade.go
@@ -16,7 +16,7 @@ import (
 	"github.com/dustin/go-humanize"
 	"github.com/mitchellh/go-homedir"
 	cp "github.com/otiai10/copy"
-	"github.com/shirou/gopsutil/disk"
+	"github.com/shirou/gopsutil/v4/disk"
 	"github.com/syndtr/goleveldb/leveldb"
 	"github.com/syndtr/goleveldb/leveldb/util"
 

--- a/go.mod
+++ b/go.mod
@@ -26,7 +26,6 @@ require (
 	github.com/phayes/freeport v0.0.0-20220201140144-74d24b5ae9f5
 	github.com/prometheus/client_golang v1.19.1
 	github.com/sethvargo/go-retry v0.3.0
-	github.com/shirou/gopsutil v3.21.4-0.20210419000835-c7a38de76ee5+incompatible
 	github.com/shirou/gopsutil/v4 v4.25.3
 	github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 	github.com/testcontainers/testcontainers-go v0.32.0
@@ -37,7 +36,6 @@ require (
 	github.com/Azure/go-ansiterm v0.0.0-20230124172434-306776ec8161 // indirect
 	github.com/Microsoft/go-winio v0.6.2 // indirect
 	github.com/Microsoft/hcsshim v0.11.5 // indirect
-	github.com/StackExchange/wmi v1.2.1 // indirect
 	github.com/aead/siphash v1.0.1 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f // indirect

